src/include/daos_srv/pool.h

```diff
@@ -190,6 +190,7 @@ int ds_pool_tgt_map_update(struct ds_pool *pool, struct pool_buf *buf,
 			   unsigned int map_version);
 
 int ds_pool_start(uuid_t uuid);
+int ds_pool_start_ex(uuid_t uuid, uint32_t flags);
```

Comment:
The correct way of writing both function definitions and functions is with the return code on a different line.  Unfortunately when you have a block like this which are all already incorrect then clang-format will also get the indentation of the modified one wrong as well.
```suggestion
int
ds_pool_start_ex(uuid_t uuid, uint32_t flags);
```

Reply:
- [ ] Fix coding style

Reply:
fixed with commit https://github.com/daos-stack/daos/commit/848c314b31e924c42492105a51c687b981900ee8